tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -366,10 +213,11 @@ def test_positive_list_all_with_per_page(self):
             {'organization-id': self.org['id'], 'per-page': per_page_count}
         )
 
-        self.assertEqual(len(lifecycle_environments), self.lces_count)
+        assert len(lifecycle_environments) == self.lces_count
         env_name_set = {env['name'] for env in lifecycle_environments}
-        self.assertEqual(env_name_set, set(self.env_names))
+        assert env_name_set == set(self.env_names)
 
+    @pytest.mark.parametrize('name', **parametrized(valid_data_list()))
```

Comment:
@latran we are not using name in this test. we need per_page values.
```suggestion
    @pytest.mark.parametrize('per_page', **parametrized([1, 5, 10]))
    @tier2
    def test_positive_list_with_pagination(self, per_page):
```

Reply:
I added that and forgot to removed it, sorry about that.